File: Error Handling and Concurrency.java
try {
    int result = 10 / 0;
} catch (ArithmeticException e) {
    System.out.println("Cannot divide by zero");
} finally {
    System.out.println("Execution finished");
}

Thread thread = new Thread(() -> System.out.println("Hello from thread!"));
thread.start();

File: Error Handling and Concurrency.py
try:
    result = 10 / 0
except ZeroDivisionError:
    print("Cannot divide by zero")
finally:
    print("Execution finished")

import asyncio

async def say_hello():
    await asyncio.sleep(1)
    print("Hello from async!")

asyncio.run(say_hello())
